hibernate-core/src/test/java/org/hibernate/jpa/test/jointable/ManyToOneJoinTableTest.java

```diff
@@ -24,11 +45,19 @@
 	protected Class<?>[] getAnnotatedClasses() {
 		return new Class[] {
 				Person.class,
-				Address.class
+				Address.class,
+				ResourceImpl.class,
+				IssuerImpl.class
 		};
 	}
 
-	@Test
+	@Override
+	protected void configure(Configuration configuration) {
+		super.configure(configuration);
+//		configuration.setProperty(AvailableSettings.OMIT_JOIN_OF_SUPERCLASS_TABLES, Boolean.FALSE.toString());
```

Comment:
Do we have any reason not to delete the above statement which is commented out?

Reply:
Not really, prior the to fix, it can be enabled to show that the fix is in fact a regression caused by the functionality which can be disabled by this property.